.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -29,9 +29,10 @@ Describe here what actually happened.
 This is extremely important! Providing us with a reliable way to reproduce
 a problem will expedite its solution.
 
-## RuboCop version
+## Gem versions
 
-Include the output of `rubocop -V` or `bundle exec rubocop -V` if using Bundler. Here's an example:
+Include the version of the `rubocop-rails` gem, and the output of `rubocop -V`, o
```

Comment:
Thank you for improving the issue template! Can you update the following and squash your commits into one?

```suggestion
Include the version of the `rubocop-rails` gem, and the output of `rubocop -V`, or
```